build.cake

```diff
@@ -160,137 +105,78 @@ Task("Copy-Files")
-            new ChocolateyNuSpecContent {Source = string.Concat("./../", binDir.Path.FullPath, "/Cake.Common.dll")},
-            new ChocolateyNuSpecContent {Source = string.Concat("./../", binDir.Path.FullPath, "/Cake.Common.xml")},
-            new ChocolateyNuSpecContent {Source = string.Concat("./../", binDir.Path.FullPath, "/NuGet.Core.dll")},
-            new ChocolateyNuSpecContent {Source = string.Concat("./../", binDir.Path.FullPath, "/Mono.CSharp.dll")},
-            new ChocolateyNuSpecContent {Source = string.Concat("./../", binDir.Path.FullPath, "/Autofac.dll")},
-            new ChocolateyNuSpecContent {Source = string.Concat("./../", binDir.Path.FullPath, "/LICENSE")}
-        }
-    });
+    foreach(var package in parameters.Packages.Chocolatey)
+    {
+        // Create package.
+        ChocolateyPack(package.NuspecPath, new ChocolateyPackSettings {
+            Version = parameters.Version.SemVersion,
+            ReleaseNotes = parameters.ReleaseNotes.Notes.ToArray(),
+            OutputDirectory = parameters.Paths.Directories.NugetRoot,
+            Files = parameters.Paths.ChocolateyFiles
+        });
+    }
 });
 
 Task("Create-NuGet-Packages")
     .IsDependentOn("Copy-Files")
-    .IsDependentOn("Run-GitVersion")
     .Does(() =>
 {
-    // Create Cake package.
-    NuGetPack("./nuspec/Cake.nuspec", new NuGetPackSettings {
-        Version = semVersion,
-        ReleaseNotes = releaseNotes.Notes.ToArray(),
-        BasePath = binDir,
-        OutputDirectory = nugetRoot,
-        Symbols = false,
-        NoPackageAnalysis = true
-    });
-
-    // Create Core package.
-    NuGetPack("./nuspec/Cake.Core.nuspec", new NuGetPackSettings {
-        Version = semVersion,
-        ReleaseNotes = releaseNotes.Notes.ToArray(),
-        BasePath = binDir,
-        OutputDirectory = nugetRoot,
-        Symbols = false
-    });
-
-    // Create Common package.
-    NuGetPack("./nuspec/Cake.Common.nuspec", new NuGetPackSettings {
-        Version = semVersion,
-        ReleaseNotes = releaseNotes.Notes.ToArray(),
-        BasePath = binDir,
-        OutputDirectory = nugetRoot,
-        Symbols = false
-    });
-
-    // Create Testing package.
-    NuGetPack("./nuspec/Cake.Testing.nuspec", new NuGetPackSettings {
-        Version = semVersion,
-        ReleaseNotes = releaseNotes.Notes.ToArray(),
-        BasePath = binDir,
-        OutputDirectory = nugetRoot,
-        Symbols = false
-    });
+    foreach(var package in parameters.Packages.Nuget)
+    {
+        // Create package.
+        NuGetPack(package.NuspecPath, new NuGetPackSettings {
+            Version = parameters.Version.SemVersion,
+            ReleaseNotes = parameters.ReleaseNotes.Notes.ToArray(),
+            BasePath = parameters.Paths.Directories.ArtifactsBin,
+            OutputDirectory = parameters.Paths.Directories.NugetRoot,
+            Symbols = false,
+            NoPackageAnalysis = true
+        });
+    }
 });
 
 Task("Update-AppVeyor-Build-Number")
```

Comment:
@devlead this task isn't required.  Running GitVersion with `/output buildserver` already takes care of that.